exercicios/para-casa/resolucao/classes/Bank.js

```diff
@@ -0,0 +1,46 @@
+export class Bank{
+
+    constructor(bankCode, bankName, transferTax){
+
+        this.bankCode = bankCode;
+        this.bankName = bankName;
+        this.#transferTax = transferTax;
+        this.numberOfClients = 0;
+        Bank.createdBanks.push({ bankCode: this.bankCode, numberOfClients: this.numberOfClients });
+
+    }
+  get transferTax() {
+    return this.#transferTax;
+  }
+
+  set transferTax(newTransferTax) {
+    this.#transferTax = newTransferTax;
+  }
+
+  addClient() {
+    this.numberOfClients++;
+    // Atualiza a quantidade de clientes no banco correspondente
+    const bankIndex = Bank.createdBanks.findIndex(bank => bank.bankCode === this.bankCode);
+
+    if (bankIndex !== -1) {
+
+      Bank.createdBanks[bankIndex].numberOfClients = this.numberOfClients;

```

Comment:
Legal ver como você fez para adicionar mais um em numberOfClients, no meu exercicício aqui eu fiz que o numberOfClients += 1, pra toda ver que entrar na função de addClient adicionar mais 1 ao valor.
É interessantes ver outras soluções como a sua :)

Reply:
Então, eu ia fazer com +=1, mas fiz dessa maneira que achei na internet e deu certo tbm... to experimentando coisas novas :)